sql_metadb/derived_tables/finance_invoice_transactions.sql

```diff
@@ -0,0 +1,136 @@
+),
+finance AS (
+    SELECT
+        fiscal_year.id AS fiscal_year_id,
+        jsonb_extract_path_text(fiscal_year.jsonb, 'code') AS fiscal_year,
+        ledger.id AS ledger_id,
+        jsonb_extract_path_text(ledger.jsonb, 'name') AS ledger_name,
+        budget.id AS budget_id,
+        jsonb_extract_path_text(budget.jsonb, 'name') AS budget_name,
+        fund.id AS fund_id,
+        jsonb_extract_path_text(fund.jsonb, 'code') AS fund_code
+    FROM
+        folio_finance.fiscal_year AS fiscal_year
+        LEFT JOIN folio_finance.ledger AS ledger ON ledger.fiscalyearoneid = fiscal_year.id
+        LEFT JOIN folio_finance.fund AS fund ON fund.ledgerid = ledger.id
+        LEFT JOIN folio_finance.budget AS budget ON budget.fundid = fund.id
+            AND budget.fiscalyearid = fiscal_year.id
+),
+transactions AS (
+    SELECT
+        transaction.id AS transaction_id,
+        jsonb_extract_path_text(transaction.jsonb, 'fromFundId')::uuid AS fromFundId,
+        jsonb_extract_path_text(transaction.jsonb, 'amount')::numeric(19, 4) AS transaction_amount,
+        jsonb_extract_path_text(transaction.jsonb, 'currency') AS transaction_currency,
+        jsonb_extract_path_text(transaction.jsonb, 'sourceInvoiceLineId')::uuid AS sourceInvoiceLineId
+    FROM
+        folio_finance.transaction AS transaction
+)
+SELECT
+    invoice.invoice_id,
+    invoice.folio_invoice_number,
+    invoice.vendor_id,
+    invoice.vendor_name,
+    invoice.vendor_invoice_number,
+    invoice.invoice_date,
+    invoice.invoice_status,
+    invoice.invoice_line_id,
+    invoice.invoice_line_total,
+    invoice.invoice_currency,
+    invoice.invoice_line_distribution_value,
+    invoice.invoice_line_distribution_type,
+    finance_invoice.fiscal_year_id AS invoice_line_fiscal_year_id,
+    finance_invoice.fiscal_year AS invoice_line_fiscal_year,
+    finance_invoice.ledger_id AS invoice_line_ledger_id,
+    finance_invoice.ledger_name AS invoice_line_ledger_name,
+    finance_invoice.budget_id AS invoice_line_budget_id,
+    finance_invoice.budget_name AS invoice_line_budget_name,
+    finance_invoice.fund_id AS invoice_line_fund_id,
+    finance_invoice.fund_code AS invoice_line_fund_code,
+    invoice.invoice_exchange_rate,
+    transactions_invoice.transaction_id AS invoice_line_transaction_id,
+    transactions_invoice.transaction_amount AS invoice_line_transaction_amount,
+    transactions_invoice.transaction_currency AS invoice_line_transaction_currency
+FROM
+    invoice
+    LEFT JOIN finance AS finance_invoice ON finance_invoice.fund_id = invoice.invoice_line_fund_id
+    LEFT JOIN transactions AS transactions_invoice ON transactions_invoice.sourceInvoiceLineId = invoice.invoice_line_id
+        AND transactions_invoice.fromFundId = invoice.invoice_line_fund_id
+    ORDER BY
+        invoice_id,
+        invoice_line_id;
```

Comment:
These rows can be deleted because a derived table does not need sorting. I heard that from the reviewer during my last PR and we should implement it that way.